settings.gradle

```diff
@@ -40,3 +40,4 @@ include 'docs-website'
 include 'metadata-models-custom'
 include 'entity-registry:custom-test-model'
 include 'spark-lineage'
+include 'metadata-integration:java:datahub-client'
```

Comment:
Why is there 3 levels of package here? There should be just 2. 

Alternatively we can just have a top-level "datahub-client" package

Reply:
Keeping it as is considering slack discussion